public/apps/configuration/utils/test/password-edit-panel.test.tsx

```diff
@@ -24,21 +39,43 @@ describe('Password edit panel', () => {
   const updateIsInvalid = jest.fn();
   const useState = jest.spyOn(React, 'useState');
   const useEffect = jest.spyOn(React, 'useEffect');
+  const mockCoreStart = {
+    http: 1,
+  };
 
   beforeEach(() => {
     useEffect.mockImplementationOnce((f) => f());
     useState.mockImplementation((initialValue) => [initialValue, setState]);
-    component = shallow(
-      <PasswordEditPanel updatePassword={updatePassword} updateIsInvalid={updateIsInvalid} />
-    );
   });
 
-  it('renders', () => {
-    expect(updatePassword).toHaveBeenCalledTimes(1);
-    expect(updateIsInvalid).toHaveBeenCalledTimes(1);
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('renders', (done) => {
+    mount(
```

Comment:
qq: Compare to the original code why we need this mount for a full rendering here?

Reply:
Because `React.useEffect()` is not called using `shallow()` and this test is testing functionality in that hook